gravitee-reporter-elasticsearch-http/src/main/java/io/gravitee/reporter/elastic/config/ElasticConfiguration.java

```diff
@@ -115,6 +134,13 @@ public String getIndexName() {
 		return endpoints;
 	}
 
+    private List<String> initializeiIngestPlugins() {
+        String ingestPluginsSt = environment.getProperty("reporters.elasticsearch.ingestPlugins", String.class);
```

Comment:
Not sure `ingestPlugins`is the better property name.
I think about something like:
```yml
  elasticsearch:
    enabled: true # Is the reporter enabled or not (default to true)
    endpoints:
      - http://${ds.elastic.host}:${ds.elastic.port}
    plugins:
        ingest: geop, user_agent
```

WDYT ?

Reply:
I'm aggree with you. It's better plugins.ingest